JsonSchema/JsonSchema.cs

```diff
@@ -510,6 +512,8 @@ private static void PopulateBaseUris(JsonSchema schema, JsonSchema resourceRoot,
 		}
 	}
 
+	private static bool IsDefinedSpecVersion(SpecVersion specVersion) => _specVersions.Contains(specVersion);
```

Comment:
I think I'd prefer not encapsulating this in a method since it's so simple. Just having the `.Contains()` call in the `if` statement reads fine, and it's not used in multiple places.
Maybe renaming `_specVersions` to `_definedSpecVersions` gets the message across.